docs/guides/server/hostname.adoc

```diff
@@ -85,42 +85,276 @@ To reduce attack surface, the administration endpoints for Keycloak and the Admi
 Therefore, you can secure them by using a reverse proxy.
 For more information about which paths to expose using a reverse proxy, see <@links.server id="reverseproxy"/>.
 
-== Example Scenarios
-The following are more example scenarios and the corresponding commands for setting up a hostname.
 
-Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+== Configuration Examples by Different Scenarios
 
-=== Exposing the server behind a TLS termination proxy
 
-In this example, the server is running behind a TLS termination proxy and publicly available from `https://mykeycloak`.
+Through this set of hostname configuration options, Keycloak aims to provide a flexible way to adapt to the characteristics of different environments.
 
-.Configuration:
-<@kc.start parameters="--hostname=mykeycloak --proxy=edge"/>
+Misconfiguration of the Hostname and/or its Reverse Proxy options can lead to unusual and unpleasant behavior, such as Keycloak's Administration Console not loading correctly.
 
-=== Exposing the server without a proxy
+In an attempt to further improve the user experience and minimize the occurrence of misconfigurations, see the list of sample Hostname and Proxy configurations below, which is oriented to possible scenarios that may exist in different computing environments.
 
-In this example, the server is running without a proxy and exposed using a URL using HTTPS.
+The following are more example scenarios and the corresponding commands for setting up the `Hostname` and/or `Proxy` options.
 
-.Keycloak configuration:
-<@kc.start parameters="--hostname-url=https://mykeycloak"/>
+> Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+
+
+=== Scenario 01 - Beginner/Quick Test in Localhost with HTTP only
+
+* __**Description**__:
+  ** The most basic option to run, in localhost, all default values.
+* __**Recommended Settings / Keycloak configuration**__:
+
+  <@kc.startdev parameters="" />
+
+* __**Expected Behavior**__:
+  ** Both Frontend/Backend and Admin Console at: http://localhost:8080
+
+---
+
```

Comment:
I think you misunderstood which lines I meant (see the lines [#116](https://github.com/keycloak/keycloak/compare/2eba37955ddf45c6c9417b8b2aa0dd45d16cc8e0..a9d0891d8c0659c85390dc73c2b338d5cb40989e#diff-4b01f529c307cfbbe1aecbbc849b4d8906935c34fc732a4fccea29457d1dd47fL116), [#130](https://github.com/keycloak/keycloak/compare/2eba37955ddf45c6c9417b8b2aa0dd45d16cc8e0..a9d0891d8c0659c85390dc73c2b338d5cb40989e#diff-4b01f529c307cfbbe1aecbbc849b4d8906935c34fc732a4fccea29457d1dd47fL130), etc.), because you removed them in your last update like I wanted it and the output is just fine. 😊 

The thing is that although I in my previous answer I just commented one specific line no. 115, the GH UI always shows it as a range of lines. Therefore you need to always look at the very last line of the green range, unless there is  a note like `Comment on lines 103 to 113` above it.